users/users-model.js

```diff
@@ -0,0 +1,25 @@
+const db = require('../database/dbConfig');
+
+module.exports = {
+  find,
+  findBy,
+  findById,
+  addUser,
+}
+
+function find() {
+  return db('users').select('id', 'username', 'password');
```

Comment:
Why use a `select` for all fields here?
`db(tableName)` gets all fields by default.